### DISCUSSION TITLE
I cannot carry out the current vote

### DISCUSSION THREAD

**Discussion by KDK-Port** [iotaledger/firefly#3378] [Help & Support]
When I am asked to enter my password, the error message comes: "An error has occurred". I then tried to send my IOTA credit to another address, then the same error message comes after entering the password. What can I do so that these functions can be carried out?
Thank you in advance for your message.

**Comment by begonaalvarezd** [comment]
Hello, can you try creating a new profile and see if the problem persists?

You can either export your current stronghold (settings >  security > export backup) and use it to create a new profile (add new profile > migrate or restore wallet > i have a file backup > drop your strongold and follow the steps) or create a new profile directly from your mnemonic (addd new profile > migrate or restore wallet > i have a firefly recovery phrase > enter your 24 words and follow steps) 

🙏🏼
